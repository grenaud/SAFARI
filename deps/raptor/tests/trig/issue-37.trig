<x> { [ a <test> ] <shows> <bug> .}
