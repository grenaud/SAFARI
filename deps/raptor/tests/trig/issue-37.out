_:genid1 <http://example.librdf.org/shows> <http://example.librdf.org/bug> <http://example.librdf.org/x> .
_:genid1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://example.librdf.org/test> <http://example.librdf.org/x> .
